lib/ace/autocomplete.js

```diff
@@ -434,7 +434,7 @@ var FilteredList = function(array, filterText) {
         this.filterText = str;
         matches = this.filterCompletions(matches, this.filterText);
         matches = matches.sort(function(a, b) {
-            return b.exactMatch - a.exactMatch || b.score - a.score;
+            return b.exactMatch - a.exactMatch || a.meta > b.meta || b.score - a.score;
```

Comment:
pls revert the direction: `b.meta < a.meta` so that it looked consistent with subsequent score difference